onedal/_device_offload.py

```diff
@@ -153,11 +165,36 @@ def _get_host_inputs(*args, **kwargs):
     return q, hostargs, hostkwargs
 
 
-def _extract_usm_iface(*args, **kwargs):
+def _extract_array_attr(*args, **kwargs):
+    """Extracts USM iface, array namespace and Hardware device
+    the array data resides on.
+
+    Returns
+    -------
+    usm_iface : Dict
+        SUA protocol dictionary describing the array.
```

Comment:
What is SUA?

Reply:
Yeah, make sense to clarify abbreviation